docs/starters.yml

```
@@ -2056,3 +2056,13 @@
     - Uses styled-components + styled-system
     - SEO with Sitemap, Schema.org JSONLD, Tags
     - Responsive images with gatsby-image
+- url: https://amazing-jones-e61bda.netlify.app/
+  repo: https://github.com/WebCu/gatsby-material-kit-react
+  description: Adaptation of Material Kit React to Gatsby
+  tags:
+    - Material Kit React
+    - Material UI
```

```suggestion
    - Styling:Material
```

We have a whitelisted set of tags that are valid for starters, you can see them here: https://github.com/gatsbyjs/gatsby/blob/master/docs/categories.yml

^ this list is used so the Starter showcase doesn't get bloated with all sorts of tags we can't manage, please update these to tags included in that file under `starters:` 🙂